core/tauri/src/webview/webview_window.rs

```diff
@@ -1219,6 +1208,21 @@ impl<R: Runtime> WebviewWindow<R> {
   }
 }
 
+/// Desktop window getters.
+#[cfg(desktop)]
+impl<R: Runtime> WebviewWindow<R> {
+  /// Get the cursor position  relative to the top-left hand corner of the desktop.
+  ///
+  /// Note that the top-left hand corner of the desktop is not necessarily the same as the screen.
+  /// If the user uses a desktop with multiple monitors,
+  /// the top-left hand corner of the desktop is the top-left hand corner of the monitor at the top-left of the desktop.
+  ///
+  /// The coordinates can be negative if the top-left hand corner of the window is outside of the visible screen region.
```

Comment:
At least on windows 0,0 will be on the top left of the _primary_ monitor, so if you have monitors to the left of your main one (like i do), they will also have negative coordinates while being in the visible screen region.

Reply:
this was copied from `tao` which is copied from `winit` ofc, maybe need to adjust the wording a bit, maybe just simplify it to:
> Get the cursor position relative to the top-left hand corner of the main monitor.

Reply:
Sorry for the late reply (out of office), but iirc the comment was correct for x11 == it's always a positive number on x11.

No idea about macos.

I can test both tomorrow when i'm back.

Reply:
Mac has negative coordinates as well. Just like Windows, it relates to the main monitor.

Reply:
sorry for the delay. X11 indeed starts with 0,0 on the leftmost monitor regardless of whether it's the primary one or not.